lib/compat/wordpress-6.9/command-palette.php

```diff
@@ -3,14 +3,78 @@
  * Enqueues the assets required for the Command Palette.
  */
 function gutenberg_enqueue_command_palette_assets() {
-	if ( ! is_admin() ) {
-		return;
+	global $menu, $submenu;
+
+	$command_palette_settings = array();
+
+	if ( $menu ) {
+		$menu_commands = array();
+		foreach ( $menu as $menu_item ) {
+			if ( empty( $menu_item[0] ) || ! empty( $menu_item[1] ) && ! current_user_can( $menu_item[1] ) ) {
+				continue;
+			}
+
+			// Remove all HTML tags and their contents.
+			$menu_label = $menu_item[0];
+			while ( preg_match( '/<[^>]*>/', $menu_label ) ) {
+				$menu_label = preg_replace( '/<[^>]*>.*?<\/[^>]*>|<[^>]*\/>|<[^>]*>/s', '', $menu_label );
+			}
+			$menu_label = trim( $menu_label );
```

Comment:
This process was a bit tedious. For example, the "Comments" menu is actually made up of the following HTML:

<img width="182" height="69" alt="Image" src="https://github.com/user-attachments/assets/80031967-b934-467f-966b-4d3bfee2147d" />

```html
Comments <span class="awaiting-mod count-0"><span class="pending-count" aria-hidden="true">0</span><span class="comments-in-moderation-text screen-reader-text">0 Comments in moderation</span></span>
```
The command palette cannot handle labels that contain HTML, so I had to remove the HTML **and the content within it**.